.github/workflows/docker.yml

```
@@ -0,0 +1,33 @@
+name: Docker
+on:
+  - push
+  - pull_request
+jobs:
+  test:
+    name: Docker
+    runs-on: ubuntu-20.04
+    steps:
+    - name: Checkout source
+      uses: actions/checkout@v1
+    - name: Poke config
+      run: |
+        cp config/example.storage.yml config/storage.yml
+        cp config/docker.database.yml config/database.yml
+        touch config/settings.local.yml
+    - name: Build Docker Image
+      run: |
+        docker-compose build
+    - name: Start Docker-Compose
+      run: |
+        docker-compose up -d
+        sleep 15 # let the DB warm up a little
+    - name: Prepare Database
+      run: |
+        docker-compose run --rm web rake db:migrate
```

```suggestion
        docker-compose run --rm web rails db:migrate
```

Since we are on a fairly new version of Rails, this is the same and looks a bit nicer and is more up to date. 

(I will comment only this one random instance, but there are a few of them)

---

I tested locally and this seems to work well! I’m hesitant to apply this change since we seem to have the “rake” and “Rakefile” naming convention throughout the repo. Maybe we can change this globally in another PR?